README.md

```diff
@@ -53,6 +53,8 @@ use):
   classifications.
 * [_Reporter_](#reporter) - presents results in various formats such as visual reports, Open Source notices or
   Bill-Of-Materials (BOMs) to easily identify dependencies, licenses, copyrights or policy rule violations.
+* [_Notifier_](#notifier) - notifies the user for found issues or license / copyright findings based on a customizable
```

Comment:
IMO we should not explicitly say what *about* the notifier notifies; it's the whole point that this is completely customizable, but just give examples.

Like: "sends status notifications (like about issues in ORT result files) to arbitrary receivers."